INSTALL.md

```diff
@@ -103,8 +108,258 @@ docker container stop elasticsearch_test; docker container rm elasticsearch_test
 ```
 
 The following command starts a local MongoDB instance, runs the test suite, then stops and deletes the containers:
+
 ```shell
 docker run -d --name mongo_test -p 27017:27017 -d mongo:4.4.6 \
 && OPTIMADE_DATABASE_BACKEND="mongodb" py.test; \
 docker container stop mongo_test; docker container rm mongo_test
 ```
+
+## Container image
+
+### Retrieve the image
+
+The [`optimade` container image](https://github.com/Materials-Consortia/optimade-python-tools/pkgs/container/optimade) is available from the [GitHub Container registry](https://ghcr.io).
+To pull the latest version using [Docker](https://docs.docker.com/) run the following:
+
+```shell
+docker pull ghcr.io/materials-consortia/optimade:latest
```

Comment:
When I try to execute this line (nr. 126) with SUDO, I get the error: 

Error response from daemon: Head "https://ghcr.io/v2/materials-consortia/optimade/manifests/latest": unauthorized

I think the package is currently still set to private and we need to make it public.

Reply:
Yes, that's correct. This will happen when this PR is merged. See also [my comment above](https://github.com/Materials-Consortia/optimade-python-tools/pull/1171#issuecomment-1126167462).

Reply:
For now you can test it by logging in with a password that is a PAT (personal access token) with read permissions to packages :) See more [here](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry)